syscall/syscall_stublookup.c

```diff
@@ -189,13 +189,9 @@ uintptr_t STUB_nx_vsyslog(int nbr, uintptr_t parm1, uintptr_t parm2,
  */
 
 uintptr_t STUB_close(int nbr, uintptr_t parm1);
-#ifdef CONFIG_LIBC_IOCTL_VARIADIC
-uintptr_t STUB_fs_ioctl(int nbr, uintptr_t parm1, uintptr_t parm2,
-            uintptr_t parm3);
-#else
 uintptr_t STUB_ioctl(int nbr, uintptr_t parm1, uintptr_t parm2,
-            uintptr_t parm3);
-#endif
+            uintptr_t parm3, uintptr_t parm4, uintptr_t parm5,
+            uintptr_t parm6);
```

Comment:
This is terrible.  I am very strongly opposed to this and will not merge this PR with this kind of code.  Very bad.

Please follow the model of syslog, vsyslog, and nx_vsyslog.  That will work.  This is bad.

Reply:
It is amateurish and inefficient and the worst solution to the problem that I could possible imagine.

Reply:
But, I just follow open/fcntl/sem_open/mq_open/prctl approach. Both method(open .v.s. syslog) work correctly, I select the current one just because ioctl is more like these functions than syslog. I guess that syslog use a different way just because syslog may pass arguments more than 6, but other just need one.

Reply:
The all of those are wrong too.

I will not merge this PR with that implementation.